### ISSUE TITLE
Force a version

### ISSUE THREAD

**Issue by Firesphere** [Firesphere/silverstripe-solr-search#267]
**Is your feature request related to a problem? Please describe.**

It's possible the version can't be detected, or at runtime, the version isn't available (yet). I want to set the version manually

**Describe the solution you'd like**

Add a `solr_version: 9` option to the YML

**Comment by phptek** [comment]
Came here looking for this. It seems whatever version in-use (and returned by the Solr server) that the Solr9+ template is used:

```
<?xml version="1.0" encoding="UTF-8" ?>
<!-- Solr 9+ Implementation -->
<!--
...
```

**Comment by Firesphere** [comment]
This should be fixed in release 2.0.1 => https://codeberg.org/Firesphere/silverstripe-solr
